src/ResourceManager/AzureBatch/Commands.Batch/Jobs/NewBatchJobCommand.cs

```diff
@@ -30,6 +30,7 @@ public class NewBatchJobCommand : BatchObjectModelCmdletBase
 
         [Parameter]
         [ValidateNotNullOrEmpty]
+        [Alias("CommonEnvironmentSetting")]
```

Comment:
Note: the Alias here is not necessary

Reply:
Indeed... but it follows the pattern you wanted of having non-plural names for things.

Would you prefer I remove this and leave it plural...?